Components/Image/image.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
You can add meta descriptions to personalise more.. 
like
<meta name="author" content="Mohit">
    <meta name="description"
        content="about your ui">

Reply:
Noted the suggestion in notion page. Would implement it later after submission though as i am having less time now due to college work.